rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -534,6 +535,9 @@ def _build_cluster(ocm_cmnd, rosa_cmnd, cluster_name_seed, must_gather_all, prov
     if operator_roles_prefix:
         cluster_cmd.append("--operator-roles-prefix")
         cluster_cmd.append(cluster_name_seed)
+    if args.ticket_id:
+        jira_apis.verify_issue_id(args.ticket_id)
+        cluster_cmd.append("--tags=TicketId:{}.format(args.ticket_id")
```

Comment:
we need to add here some check, so if ticket id exists, we need to verify the result of verify_issue_id before appending the tag to the cluster_cmd

Reply:
> we need to add here some check, so if ticket id exists, we need to verify the result of verify_issue_id before appending the tag to the cluster_cmd

We have a check [here](https://github.com/cloud-bulldozer/perfscale-managed-services/blob/6324f5b16460f6d28a85fb146ea6acfccbb9f5a2/rosa-hypershift/utils/jira_apis.py#L55), in case of invalid Ticket-Id, it will exit.